### PR TITLE
Add kudu6

### PR DESCRIPTION
**Pull request by jacobgkau**
Adds the Kudu 6 to tech docs.

### CODE REVIEW

**Review by thomas-zimmerman — COMMENTED** [review]
If this system is a switchable graphics system, I would like to have it called out in the GPU section with the on CPU AGPU model information listed. Everything else looks good to me.

**Comment by jacobgkau** [comment]
> If this system is a switchable graphics system, I would like to have it called out in the GPU section with the on CPU AGPU model information listed. Everything else looks good to me.

No other switchable graphics systems on Tech Docs include that information (except for galp5, which refers to its non-switchable version), but it can be added.

**Comment by cwsystem76** [comment]
The README file says that the kudu6 chipset is an Intel HM570, which I'm pretty sure is not correct.

File: src/models/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ This documentation includes the following System76 models:
 - [Darter Pro (darp7)](darp7/README.md)
 - [Galago Pro (galp5)](galp5/README.md)
 - [Gazelle (gaze16)](gaze16/README.md)
+- [Kudu (kudu6)](models/kudu6/README.md)
```
Comment (leviport) [inline src/models/README.md:13]:
```suggestion
+ [Kudu (kudu6)](kudu6/README.md)
```

Comment (jacobgkau) [reply src/models/README.md]: Fixed with f8c20f5, thank you for catching that.

**Review by leviport — APPROVED** [review]
Looks great!

**Pull request merged**